README.md

```diff
@@ -542,6 +542,34 @@ showCard(Pair('X', '♠'));
 // ! TypeError: ‘showCard’ expected a value of type (Pair Rank Suit) as its first argument; received Pair("X", "♠")
 ```
 
+#### `EnumType`
+
+`EnumType` is used to construct [enumerated types][5].
+
+```javascript
+//    TimeUnit :: Type
+const TimeUnit = $.Enum(['milliseconds', 'seconds']);
+
+const def = $.create($.env.concat([TimeUnit, $.ValidDate, $.ValidNumber]));
+
+//    convertTo :: TimeUnit -> ValidDate -> ValidNumber
+const convertTo =
+def('convertTo',
+    {},
+    [TimeUnit, $.ValidDate, $.ValidNumber],
+    (unit, date) => date.valueOf() * (unit === 'milliseconds' ? 1000 : 1));
+    
+convertTo('milliseconds', new Date(1));
+// => 1000
+
+convertTo('days', new Date(1));
+// => ! TypeError: ‘convertTo’ expected a value of type ("milliseconds" | "seconds") as its first argument; received "days"
+```
+
+```haskell
+$.EnumType :: [Any] -> Type
+```
```

Comment:
This should precede the example, as with the `$.RecordType` documentation. Also, we're a bit inconsistent: we don't include the `$.` prefix in the type constructor signatures. To be consistently inconsistent, let's drop the prefix. :)